site/docs/hosting/cloudflare-workers.md

```diff
@@ -0,0 +1,121 @@
+# Hosting: Cloudflare Workers
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is a public serverless computing platform that offers a convenient and simple solution for running JavaScript at the edge.
+With the ability to handle HTTP traffic and work with [Service Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), building Telegram bots becomes a breeze.
```

Comment:
```suggestion
Having the ability to handle HTTP traffic and being based [Service Worker API](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), building Telegram bots becomes a breeze.
```

Reply:
Being based Service Worker API?

Reply:
Yes, it's based on that. What's the question?

Reply:
_Having the ability to handle HTTP traffic and **being based Service Worker API**,_

Aren’t you missing a preposition or something?

Reply:
Oh. I misread all the time. Yes, it should be _being based on the Service Worker API_